source/manual/email-troubleshooting.html.md

```diff
@@ -31,8 +31,6 @@ will take you to a form where you can search by email address.
 
 A courtesy copy of every email sent from integration, staging and production
```

Comment:
Out-of-scope: we should kill this doc (it's too general to be useful) and migrate it's content elsewhere:

- The link in the first section is in other docs.

- The second section appears to be covered [here](https://docs.publishing.service.gov.uk/manual/govuk-notify.html).

- The third section is much more relevant [here](https://docs.publishing.service.gov.uk/manual/email-notifications-how-they-work.html).

Reply:
Sure, I'll see if I can take that on later in the week.